.pre-commit-config.yaml

```diff
@@ -38,13 +35,13 @@ repos:
   - repo: https://github.com/akaihola/darker
     rev: v2.1.1
     hooks:
-    -   id: darker
-        args: ["--check"]
+      - id: darker
+        args: ["--check", "--diff", "--color"]
```

Comment:
We don't want "--diff" for the pre-commit, because `--diff` only shows the recommended update but does not update the file, and we want the pre-commit to update the file.

```suggestion
        args: ["--check", "--color"]
```

Reply:
According to https://github.com/akaihola/darker the `--check` already does not update the file, right?
And, in that case, if writing to the files directly was intended. Both `--check` and. `--diff` needs to be removed, I guess.